hardware/health/LearnedCapacityBackupRestore.cpp

```diff
@@ -74,12 +75,7 @@ void LearnedCapacityBackupRestore::ReadFromPersistStorage() {
     }
 
     buffer = ::android::base::Trim(buffer);
-    try {
-        persist_capacity = std::stoi(buffer);
-    } catch (std::out_of_range &e) {
-        LOG(WARNING) << "Battery capacity in persist storage file is out of bounds: " << buffer;
-        return;
-    } catch (std::invalid_argument &e) {
+    if (!android::base::ParseInt(buffer.c_str(), &persist_capacity)) {
```

Comment:
Note that `Trim()` above goes from the `::android` namespace. Not that it makes too big of a difference.

Reply:
Indeed. I wasn't sure which to use as below there is from `android::base::`